spec.html

```diff
@@ -33039,18 +33039,18 @@ <h1>Properties of Array Iterator Instances</h1>
             </tr>
             <tr>
               <td>
-                [[IteratedObject]]
+                [[IteratedArray]]
```

Comment:
The value of this internal slot is not necessarily an Array exotic object, yes? It may be best to leave this as `IteratedObject`.